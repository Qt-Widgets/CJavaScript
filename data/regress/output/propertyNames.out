{ E: 2.71828, LN2: 0.693147, LN10: 2.30259, PI: 3.14159, SQRT2: 1.41421, abs: double abs(double), acos: double acos(double), asin: double asin(double), atan: double atan(double), atan2: double atan2(double, double), ceil: double ceil(double), cos: double cos(double), exp: double exp(double), floor: double floor(double), log: double log(double), pow: double pow(double, double), round: double round(double), sin: double sin(double), sqrt: double sqrt(double), tan: double tan(double) }
[ 'E', 'LN2', 'LN10', 'PI', 'SQRT2', 'abs', 'acos', 'asin', 'atan', 'atan2', 'ceil', 'cos', 'exp', 'floor', 'log', 'pow', 'round', 'sin', 'sqrt', 'tan' ]
