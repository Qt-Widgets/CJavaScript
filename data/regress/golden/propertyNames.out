{}
[ 'E',
  'LN10',
  'LN2',
  'LOG2E',
  'LOG10E',
  'PI',
  'SQRT1_2',
  'SQRT2',
  'random',
  'abs',
  'acos',
  'asin',
  'atan',
  'ceil',
  'cos',
  'exp',
  'floor',
  'log',
  'round',
  'sin',
  'sqrt',
  'tan',
  'atan2',
  'pow',
  'max',
  'min' ]
